### PR TITLE
[KIECLOUD-367] - Operator Hub catalog changes

### PR DESCRIPTION
**Pull request by swati-kale**
Signed-off-by: Swati Kale <swkale@redhat.com>

### CODE REVIEW

**Comment by swati-kale** [comment]
@tchughesiv Do I need to commit "/kiecloud-operator.1.4.0.clusterserviceversion.yaml" and "businessautomation-operator.1.4.0.clusterserviceversion.yaml" too?

**Comment by swati-kale** [comment]
Please find the screen shot, changes are as below
1) New image icon
2) Name - Business Automation
3)Provider - Red Hat
4) Description on card
5) Detailed description on metadata

BA

**Comment by tchughesiv** [comment]
/ok-to-test
@swati-kale which release is this targeting?

File: tools/csv-gen/csv-gen.go [hunk 1]
```diff
@@ -150,7 +150,7 @@ func main() {
...
-		templateStruct.Spec.Description = descrip
+		templateStruct.Spec.Description = "Deploys and manages Red Hat Process Automation Manager and Red Hat Decision Manager environments.\n\n* **Red Hat Process Automation Manager** is a platform for developing containerized microservices and applications that automate business decisions and processes. It includes business process management (BPM), business rules management (BRM), and business resource optimization and complex event processing (CEP) technologies. It also includes a user experience platform to create engaging user interfaces for process and decision services with minimal coding.\n\n * **Red Hat Decision Manager** is a platform for developing containerized microservices and applications that automate business decisions. It includes business rules management, complex event processing, and resource optimization technologies. Organizations can incorporate sophisticated decision logic into line-of-business applications and quickly update underlying business rules as market conditions change.\n\n[See more](https://www.redhat.com/en/products/process-automation)."
```
Comment (tchughesiv) [inline tools/csv-gen/csv-gen.go]: maybe change to something like this so reusing `descrip` var?
```golang
templateStruct.Spec.Description = descrip + "\n\n* **Red Hat Process Automation Manager** is a platform for developing containerized microservices and applications that automate business decisions and processes. It includes business process management (BPM), business rules management (BRM), and business resource optimization and complex event processing (CEP) technologies. It also includes a user experience platform to create engaging user interfaces for process and decision services with minimal coding.\n\n * **Red Hat Decision Manager** is a platform for developing containerized microservices and applications that automate business decisions. It includes business rules management, complex event processing, and resource optimization technologies. Organizations can incorporate sophisticated decision logic into line-of-business applications and quickly update underlying business rules as market conditions change.\n\n[See more](https://www.redhat.com/en/products/process-automation)."
```

Comment (tchughesiv) [reply tools/csv-gen/csv-gen.go]: did `make csv` produce the same results?

Comment (swati-kale) [reply tools/csv-gen/csv-gen.go]: Yes, i did change createdAt: "2020-02-26 09:14:55" which I ignored.

**Comment by tchughesiv** [comment]
@swati-kale appears there are conflicts that need to be resolved first

**Comment by tchughesiv** [comment]
@swati-kale instead of a merge with master... should do `git rebase master`

**Pull request merged**